name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/FinTechGroupBankPDFExtractor.java

```diff
@@ -1491,6 +1504,17 @@ private void addDepotStatementTransaction()
                             t.setCurrencyCode(v.get("currency"));
                             t.setNote(trim(v.get("note")));
                         })
+                        .section("note2").optional() //
+                        .match("^[\\s]+(?<note2>ZINSPILOT Auszahlung .*)$")
+                        .assign((t, v) -> {
+                            t.setNote(trim(v.get("note2")));
+                            t.setType(AccountTransaction.Type.INTEREST);
+                        })
+                        .section("note2").optional() //
+                        .match("^[\\s]+(?<note2> .*\\-Aktion).*$")
+                        .assign((t, v) -> {
+                            t.setNote(t.getNote() + (v.get("note2")));
+                        })
```

Comment:
```suggestion
                        .section("note2").optional() //
                        .match("^[\\s]+(?<note2> .*\\-Aktion).*$")
                        .assign((t, v) -> t.setNote(t.getNote() + (v.get("note2"))))
```

Additionally:
Wouldn't it make more sense to shorten this note a little?
`hasNote("Prämie für die Teilnahme an der Morgan Stanley-Aktion")`

Reply:
OK, I shortened it to "Prämie Morgan Stanley-Aktion".